dataBase/DBCreator.py

```diff
@@ -61,22 +61,38 @@ def __init__(self, table_name, schema ):
     
     def create( self, cursor):
         cursor.execute('create table %s (%s)' % ( self.table_name, self.schema ))
+        
+    def delete( self, cursor):
+        cursor.execute('drop table if exists %s' % ( self.table_name ))
 
 
 def main():
-    usage = 'usage: %prog [options] dbname'
-    parser = OptionParser(usage=usage)
-    (options, args) = parser.parse_args()
-    
-    if len(args) != 1:
-        parser.print_help()
-        sys.exit(1)
+	usage = 'usage: %prog [options] dbname'
+	parser = OptionParser(usage=usage)
+	parser.add_option("-l", "--lial", action="store_true", dest="list", default = False, help= "list all tables")
+	parser.add_option("-f", "--force", action="store_true", dest="force", default = False, help= "force remove and create tables")
+	parser.add_option("-t", "--table", type="string", help="name of the table", dest="table")
+	(options, args) = parser.parse_args()
+	if options.list:    
+		print "Tables in schemas are: " + str(Schemas.keys())
+		sys.exit(1)
+	if len(args) != 1:
+		parser.print_help()
+		sys.exit(1)
 
-    dbname = args[0]
-    
-    with sql.connect( dbname ) as conn:
-        for table_name, schema in Schemas.items():
-            SqlTable( table_name, schema ).create(conn)
+	dbname = args[0]
+	
+	
+	with sql.connect( dbname ) as conn:
+		for table_name, schema in Schemas.items():
+			if options.table:
+				if options.table == table_name:
+					SqlTable( table_name, schema ).create(conn)
+			elif options.force:
```

Comment:
You don't need to create SqlTable object multiple times. Just one time at line 87 is good. You can then use variable to reference it.